readme.md

```diff
@@ -21,8 +21,8 @@ Follow
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://tank-igorevich.github.io/layout_stars/)
+- [TEST REPORT LINK](https://tank-igorevich.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="986" alt="image" src="https://github.com/user-attachments/assets/4eb8de5b-2bf5-4f3c-b32d-4611ce5b2512">

Reply:
Not fixed, you need to add the demo and test reports link to the PR description 
<img width="976" alt="image" src="https://github.com/user-attachments/assets/e338c391-ec3f-4aeb-8cba-3ef74291d897">